src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <!-- Personal information group -->
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname: 
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name: 
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            checked
+            required
+          >
+        </label>
+      </fieldset>
+      <!-- Registration gtoup-->
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Pasword:
+          <input
+            type="password"
+            minlength="5"
+          >
+        </label>
+      </fieldset>
+      <!-- About you group -->
+      <fieldset class="form-group">
+        <legend>An intresting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input 
+            type="radio"
+            name="loveCats"
+            value="Yes"
+            >
+            Yes
```

Comment:
```suggestion
            <input 
              type="radio"
              name="loveCats"
              value="Yes"
            >
              Yes
```
checklist point 2 & 3